### PR TITLE
[W5][T09-3]Andrea Thniah

### PR DESCRIPTION
**Pull request by andreathniah**
Added list functionality that lists all the people in the address book after every `add` or `delete` command.

### CODE REVIEW

**Comment by josephinetanadi** [comment]
Looks good. 
Good use of incremental commits and changes!

**Comment by rachellim10** [comment]
Looks good to me! Can try making your changes as a class instead of writing all in main function.

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -4,9 +4,7 @@
 import java.util.List;
 import java.util.Optional;
 
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.CommandResult;
-import seedu.addressbook.commands.ExitCommand;
+import seedu.addressbook.commands.*;
```
Comment (stephlewyh) [inline src/seedu/addressbook/Main.java:7]: Imported classes should always be listed explicitly. E.g. `java.util.List` and not `java.util.*`, which does not follow NUS Java coding standards.

**Review by stephlewyh — COMMENTED** [review]
Hi Andrea, 

Good first try. You could update your enhancement with header method and also check out NUS Java Coding standards. 

Nevertheless, nice effort with the documentation update and comprehensive I/O test set!

**Pull request closed without merging**